pkg/types/config.go

```diff
@@ -1,6 +1,7 @@
 package types
 
 type PlatformConfig struct {
+	AuditConfig           *AuditConfig      `yaml:"auditConfig,omitempty"`
```

Comment:
I think this should be moved into the Kubernetes object

Reply:
Cool. Will do.

Reply:
AuditConfig moved into Kubernetes.
Now here:
https://github.com/flanksource/platform-cli/blob/91babc5b81de5db37b72ff5cc714a634eef2b971/pkg/types/types.go#L271-L277